internal/components/components.go

```diff
@@ -12,27 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package components
```

Comment:
Were these changes intentional?  Not sure why any component logic changes would be needed (maybe a local git issue?).

Reply:
oh right, they're in main...

Reply:
Yes that makes sense, I'll remove it and push it in a separate PR.